src/apps/slackapp/slackapp/utils.py

```diff
@@ -1,8 +1,8 @@
 from typing import Optional
 
-from hydra.utils import instantiate
-from langchain.base_language import BaseLanguageModel
-from omegaconf import OmegaConf
+from hydra.utils import instantiate  # type: ignore
```

Comment:
Reminder me what are these # type: ignore for?

Reply:
it will be removed all in all